README.md

```diff
@@ -123,36 +123,53 @@ each milestone.
 ### Next (5.0)
 
 * Improved and consistent visual style of built-in widgets.
-* Hands off approach to styling.  ipywidgets only provides an API for 
-  adding/removing CSS classes and the Style API.  The Style API provides the 
-  ability to set layout CSS traits in a mutable fashion. All other styling APIs 
+* Hands off approach to styling.  ipywidgets only provides an API for
+  adding/removing CSS classes and the Style API.  The Style API provides the
+  ability to set layout CSS traits in a mutable fashion. All other styling APIs
   are deprecated.
-* Ability to run ipywidgets outside of the notebook.
-* Initial refactoring to start to use npm for packaging the JavaScript code.
-* A new jupyter-js-widgets repository, containing all of the JS of ipywidgets.  
-  ipywidgets itself will no longer contain any JS.
-* A new interact API that behaves like Sage's equivalent.
+* Include a new interact API that behaves like Sage's equivalent.
```

Comment:
Yeah, I think it was already discussed and the changes were made https://github.com/ipython/ipywidgets/pull/288 https://github.com/ipython/ipywidgets/issues/238